charts/internal/seed-controlplane/charts/csi-driver-controller/templates/csi-driver-controller.yaml

```diff
@@ -127,7 +127,7 @@ spec:
         - --kubeconfig=/var/lib/csi-snapshotter/kubeconfig
         - --leader-election
         - --leader-election-namespace=kube-system
-        - --snapshot-name-prefix={{ .Release.Namespace }}
+        - --snapshot-name-prefix=snapshot
```

Comment:
Why is this changed? Is this backwards compatible?

Reply:
> Why is this changed?

Because while testing I hit the same length constraint as the PVs - see https://github.com/gardener/gardener-extension-provider-gcp/pull/61#discussion_r407238133.

> Is this backwards compatible?
Generally not, but in our case it is as with the current master it is not possible to create `VolumeSnapshot`s at all.

Reply:
Yes, right. Makes perfect sense. Thank you!